cmd/kava/cmd/iavlviewer/root.go

```diff
@@ -69,7 +68,7 @@ func openPrefixTree(opts ethermintserver.StartOptions, cmd *cobra.Command, prefi
 
 // ReadTree loads an iavl tree from the directory
 // If version is 0, load latest, otherwise, load named version
-// The prefix represents which iavl tree you want to read. The iaviwer will always set a prefix.
+// The prefix represents which iavl tree you want to read. The iaviewer will always set a prefix.
```

Comment:
In regards to:
https://github.com/Kava-Labs/kava/pull/1996#discussion_r1709507465

just closing the tree does not close the other underlying connections ([db](https://github.com/Kava-Labs/kava/pull/2000/files#diff-f8123781b946f87d76ac7d707941698f650588960b2ec6d70bb3dbb9f8558036R74) and dbt)